source/2019-05-09-the-ember-times-issue-97.html

```diff
@@ -42,43 +43,49 @@
 
 [Submit your talk today!](https://cfp.emberfest.eu/events/emberfest-2019)
-  
+
 ---
 
 ## [Build a GitHub Gist Clone in Octane ⛽️](https://www.youtube.com/watch?v=_JfNuppX1n8)
-<consider adding some bold to your paragraph>
-  
-In this free video tutorial, [Program with Erik](https://www.youtube.com/channel/UCshZ3rdoCLjDYuTR_RBubzw) and [Jeffrey Biles](https://courses.happyprogrammer.net/) pair program on building a simplified GitHub Gist clone in Octane. Jeffrey explains how Octane is a set of features bundled together to create a **new programming experience**. This video is geared toward beginners, folks coming from other frameworks, or people who are coming back to Ember after awhile! Share it with a friend? 
+
+In this free video tutorial, [Program with Erik](https://www.youtube.com/channel/UCshZ3rdoCLjDYuTR_RBubzw) and [Jeffrey Biles](https://courses.happyprogrammer.net/) pair program on building a simplified GitHub Gist clone in Octane. Jeffrey explains how Octane is a set of features bundled together to create a **new programming experience**. This video is geared toward beginners, folks coming from other frameworks, or people who are coming back to Ember after awhile! Share it with a friend?
 
 In addition to using the Octane blueprint, the tutorial makes use of [ember-prism](https://github.com/shipshapecode/ember-prism) and [ember-animated](https://github.com/ember-animation/ember-animated). Check out the recording and use the [GitHub repo](https://github.com/ErikCH/octane-example) as a reference to code along at home.
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
+## [Ember Event Dispatcher Clean Up for Several Mouse Events 🐁💥](https://github.com/emberjs/rfcs/pull/486)
+
+Since the early beginnings of Ember, the **Ember Event Dispatcher** handled the delegation of browser events in an application.
+But some events are easier to delegate with the Dispatcher than others. In particular, the `mouseenter`🕳 ⬅️🐁 and the `mouseleave` 🕳🐁➡️ events need to be handled differently than most other events, as they do not bubble up - a characteristic that Ember's Event Dispatcher usually relies on for event delegation.
+
+In a [new RFC](https://github.com/simonihmig/rfcs/blob/deprecate-mouseenter/text/0486-deprecate-mouseenter.md), the writer of [RFC#386](https://emberjs.github.io/rfcs/0386-remove-jquery.html) is back with a new proposal to improve the performance of jQuery-less apps in regards to event handling.
```

Comment:
```suggestion
In a [new RFC](https://github.com/simonihmig/rfcs/blob/deprecate-mouseenter/text/0486-deprecate-mouseenter.md), the writer of [RFC#386](https://emberjs.github.io/rfcs/0386-remove-jquery.html) is back with a new proposal to reduce the overhead of event handling for these rarely used event types.
```

Fwiw, jQuery also does this event trickery using `mouseover` to simulate delegated `mouseenter` internally, so the overhead is probably not only relevant for jQuery-less apps. Maybe to a lesser extend, but idk for sure...

Reply:
Thank you for the suggestion :sparkles: